node/src/actors/chain_manager/mining.rs

```diff
@@ -875,13 +873,14 @@ mod tests {
 
         // Fields required to mine a block
         let block_beacon = CheckpointBeacon::default();
+        let vrf_input = CheckpointVRF::default();
```

Comment:
could you use a CheckpointVRF that has a different hash from CheckpointBeacon, only to ensure that the test is working properly and it is not using the CheckpointBeacon instead of CheckpointVRF